src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,111 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_CAPACITY = 10;
+    private static final double GROW_INDEX = 1.5;
+    private T[] dataArray;
```

Comment:
```suggestion
    private T[] data;
```
not a mistake, but we usually don't include data type in variable's name - Java is a strongly typed language so we know it anyway